modules/articles/client/controllers/admin/list-articles.client.controller.js

```diff
@@ -10,6 +10,63 @@
   function ArticlesAdminListController(ArticlesService) {
     var vm = this;
 
-    vm.articles = ArticlesService.query();
+    vm.error = null;
+    vm.find = find;
+    vm.filter = filter;
+    vm.reset = reset;
+
+    activate();
+
+    function activate() {
+      vm.itemsPerPage = 10;
+      vm.currentPage = 1;
+      vm.filters = [];
+      vm.sorting = '-created';
```

Comment:
Why specify these here when they are only going to be plugged directly into the find() function?